modules/payloads/singles/linux/mipsbe/exec.rb

```diff
@@ -70,7 +72,9 @@ def generate(_opts = {})
     shellcode = shellcode + command_string + "\x00"
 
     # we need to align our shellcode to 4 bytes
-    (shellcode = shellcode + "\x00") while shellcode.length%4 != 0
+    while shellcode.bytesize%4 != 0
+      (shellcode = shellcode + "\x00")
```

Comment:
```suggestion
      shellcode += "\x00"
```

I don't think the parenthesis are necessary, and the short version of the *append* operator can be used :)